src/app/templates/talking-heads/talking-heads-map/talking-heads-map.component.less

```diff
@@ -1,32 +1,44 @@
-@import '../../../../common.less';
```

Comment:
why was this file changed?

Reply:
the vs code formatter change just the quotation marks from '' to ""
@import "../../../../common.less";

Reply:
Then please disable automatic formatters, and either case don't include in this pull request files that are not part of your code.